lambda/MAS/appsettings.json

```diff
@@ -21,9 +21,10 @@
   "AWS": {
     "AccessKey": "don't add this or anything else that needs securing. see here: https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?tabs=visual-studio",
     "SecretKey": "",
-    "ServiceURL": "",
+    "ServiceURL": "", 
     "BucketName": "",
-    "StaticURL": ""
+    "StaticURL": "",
+    "DistributionId": ""
```

Comment:
We can use `AWS:CloudFront:DistributionID` as there's already an Octo variable set up for that